test/unit_tests/braket/experimental/autoqasm/test_api.py

```diff
@@ -219,7 +219,7 @@ def bell_measurement_undeclared() -> None:
     return bell_measurement_undeclared
 
 
-def test_bell_measurement_undeclared(bell_measurement_undeclared) -> None:
+  def test_bell_measurement_undeclared(bell_measurement_undeclared) -> None:
```

Comment:
artifact of merge conflict resolution
```suggestion
def test_bell_measurement_undeclared(bell_measurement_undeclared) -> None:
```